Update repositories.json

CerberusX1 opened the pull request:
added The Arcadia List 

This mod list is Sim Settlement 2 based, with alot of fixes, extra quests, and enhanced graphics, while keeping this easy to mod for the end user, alot of the mods are ESL so they do not take up load slots.

this pack uses alot of 4K textures with some 2K mixed in, it is recomended to set grahpics to mediume or high, as ultra may impact preformance no matter how good your pc is, F4 is a old game and does not care how good your PC is.

tr4wzified commented:
- Download link is still pointing to halgaris helper
- Please use the tags defined here: https://github.com/wabbajack-tools/mod-lists/blob/master/allowed_tags.json
- Missing an image, requirements for the image are available on the WJ wiki

CerberusX1 commented:
> * Download link is still pointing to halgaris helper
> * Please use the tags defined here: https://github.com/wabbajack-tools/mod-lists/blob/master/allowed_tags.json
> * Missing an image, requirements for the image are available on the WJ wiki

i removed that linkk but i dont have one to replace it with, i ahve a image its in github but i cant figure out how to get the raw link to it

CerberusX1 commented:
ok i fixed the image, also i removed the link and fixed the tags, i do not currently have a download link for it unless it is supposed to be a github link as well?
